src/CSSBattle/8_ForkingCrazy/Tin.html

```diff
@@ -0,0 +1,85 @@
+<body>
+  <div class="container">
+    <div class="top">
+      <div class="stick" id="a"></div>
```

Comment:
id should be unique. if required more than once, consider using class instead

Reply:
commit 362173e